src/components/Wrap/index.tsx

```diff
@@ -1,25 +1,32 @@
 /** @jsx jsx */
 import * as React from "react";
-import { jsx, css } from "@emotion/core";
+import { jsx } from "@emotion/core";
+import { mq } from "../../../lib/utils";
```

Comment:
What was the purpose behind storing `lib` outside of the `src` directory?

Reply:
I think it was there to begin with so I just continued using it. Moving into src would make more sense

Reply:
This will be fixed with #33